### ISSUE TITLE
[RM Ch04] Instances and Instance Type Issues

### ISSUE THREAD

**Issue by pgoyal01** [cntt-n/CNTT#1255]
1. [4.2.4.1 B Instances (Basic)](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter04.md#4241-b-instances-basic)
    delete ""This instance type is intended to be available in any data centre within any operator's network" -- doesn't belong in RM

1. [4.2.4.2 N Instances (Network Intensive)](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter04.md#4242-n-instances-network-intensive)
    Suggest rewording of "This instance type is intended to be used for those applications that has high network throughput requirements (up to 50Gbps)" to "This instance type is for VNFs that have high network throughput requirements (up to 50Gbps) or for performance reasons need dedicated cores."
    delete "This instance type is more intended for VNFs and is expected to be available in regional (distributed) data centres and more towards the access networks."

1. [4.2.7.1 Naming convention](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter04.md#4271-naming-convention)
    delete "Compute Intensive" from "B/N: specifies the instance type (Basic, Network Intensive, and Compute Intensive)"
    delete "or H" from "<A ext>: specifies an optional acceleration extension for either N or H instance types."

**Comment by kedmison** [comment]
I think the editorial comment on where the NFVI profiles are expected does not contribute usefully here, so I will remove it.  

This could re-enter the RM at a later date if we choose to create some categorization of size of deployment along with expected attributes (e.g. core has x AZs and NFVI profiles a, b, c, and edge has y AZs and has only NFVI profiles a, b, etc)
